src/Playlists.tsx

```diff
@@ -28,34 +29,57 @@ export const Playlists: React.FC = () => {
 
   if (loading) {
     return (
-      <div>
+      <Box>
         <Spinner />
-      </div>
+      </Box>
     );
   }
 
   if (!data) {
     //checks to make sure data exists otherwise data needs to be data?
-    return <div>Playlist Loading from {endpoint}</div>;
+    return <Box>Playlist Loading from {endpoint}</Box>;
   }
 
-  const onClick = (options: { index: number }) => {
-    const { index } = options;
-    navigate(`/playlists/${data.items[index].id}`);
+  const onClick = (options: { key: number }) => {
```

Comment:
I would get rid of this and just do it in the map - makes it so you don't have to reaccess the data. Or the onclick should just take the playlist instead of an index.